bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/composite/AbstractCompositeModuleHandler.java

```diff
@@ -0,0 +1,79 @@
+package org.eclipse.smarthome.automation.core.internal.composite;
```

Comment:
license header and Javadoc are missing - please check code against the guidelines.